src/Android/Avalonia.Android/Platform/Storage/AndroidStorageItem.cs

```diff
@@ -392,6 +392,8 @@ public AndroidStorageFile(Activity activity, AndroidUri uri, AndroidStorageFolde
     public Task<Stream> OpenReadAsync() => Task.FromResult(OpenContentStream(Activity, Uri, false)
         ?? throw new InvalidOperationException("Failed to open content stream"));
 
+    public Stream OpenRead() => OpenReadAsync().Result;
```

Comment:
Should return `Stream?` since `OpenContentStream` can. (Not sure why `OpenReadAsync` isn't annotated but that's for another PR.)